packages/terra-clinical-result/src/resultnameheadercell/ResultNameHeaderCell.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import IconCalculator from 'terra-icon/lib/icon/IconCalculator';
+import classNames from 'classnames/bind';
+import styles from './ResultNameHeaderCell.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Content to be displayed for the Result Name row label.
+   */
+  resultName: PropTypes.string,
+  /**
+   * Content to be displayed as the supporting Unit of Measure row label.
+   */
+  unit: PropTypes.string,
+  /**
+   * The padding styling to apply to the Time Column Header Cell.
+   * One of `'none'`, `'standard'`, `'compact'`.
+   */
+  paddingStyle: PropTypes.oneOf(['none', 'standard', 'compact']),
+  /**
+   * Adds additional icon at beginning of row before the ResultName, indicating additinonal information about results or cells.
+   * One of `'none'`, `'calculated'`.
+   */
+  typeIndicator: PropTypes.oneOf(['none', 'calculated']),
+};
+
+const defaultProps = {
+  resultName: '',
+  unit: undefined,
+  paddingStyle: 'compact',
+  typeIndicator: 'none',
+};
+
+const resultRowIndicators = {
+  none: null,
+  calculated: <IconCalculator className={cx('icon-rowindicator')} />,
+};
+
+const ResultNameHeaderCell = (props) => {
+  const {
+    resultName,
+    unit,
+    paddingStyle,
+    typeIndicator,
+    ...customProps
+  } = props;
+
+  const nameHeaderCellClassnames = cx([
+    'clinical-result-name-header-cell',
+    { 'padding-standard': paddingStyle === 'standard' },
+    { 'padding-compact': paddingStyle === 'compact' },
+    customProps.className,
+  ]);
+
+  return (
+    <div {...customProps} className={nameHeaderCellClassnames}>
+      <div className={cx('name')}>
+        {typeIndicator ? resultRowIndicators[typeIndicator.toLowerCase()] : null}
```

Comment:
we can remove this check - typeIndicator's has a default value of 'none', so this will always be true.

Reply:
Removed https://github.com/cerner/terra-clinical/pull/574/commits/9eeacc2469da67d5f2d9ad3ce7cc533a407dbf13?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.scss&file-filters%5B%5D=.snap#diff-e899478378b0d5946c19eb42aeffcb5aR59